package.json

```diff
@@ -47,37 +49,57 @@
     "underscore": "1.4.4 - 1.8.3"
   },
   "devDependencies": {
-    "chai": "1.9.1",
+    "babel-core": "^5.2.17",
+    "babel-eslint": "^4.0.5",
+    "babelify": "^6.0.0",
+    "browserify": "^11.0.1",
+    "chai": "^3.2.0",
     "chai-jq": "0.0.7",
     "chai-jquery": "1.2.1",
+    "del": "^1.1.1",
     "dox": "git://github.com/jasonLaster/dox.git#marked",
+    "esperanto": "^0.7.4",
     "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
-    "grunt-contrib-clean": "0.5.0",
-    "grunt-contrib-concat": "0.1.2",
-    "grunt-contrib-connect": "0.1.2",
-    "grunt-contrib-jshint": "0.10.0",
-    "grunt-contrib-uglify": "0.2.0",
-    "grunt-contrib-watch": "0.6.1",
-    "grunt-coveralls": "0.3.0",
```

Comment:
are we dumping coveralls for code climate?  In general I find that's ok, but coveralls does sometimes warn in github on lost coverage which is nice?  Similarly we're losing plato.. which I honestly never used, but I'm sure it serves a purpose to someone?  @jasonLaster